lib/records/RecHttp.cc

```diff
@@ -57,7 +57,21 @@ const char *const TS_PROTO_TAG_UDP      = IP_PROTO_TAG_UDP.data();
 const char *const TS_PROTO_TAG_IPV4     = IP_PROTO_TAG_IPV4.data();
 const char *const TS_PROTO_TAG_IPV6     = IP_PROTO_TAG_IPV6.data();
 
-std::unordered_set<std::string_view> TSProtoTags;
+struct TSProtoTagsHash {
```

Comment:
I hate to pile on, but what is the benefit of storing pointers? A `std::string_view` isn't much more expensive to store / copy than a pointer.

Reply:
My best guess  as to the point of this code is that RecNormalizeProtoTag("tcp") is supposed to return IP_PROTO_TAG_TCP.data() aka TS_PROTO_TAG_TCP, RecNormalizeProtoTag("udp") is supposed to return IP_PROTO_TAG_UDP.data() aka TS_PROTO_TAG_UPD, etc.  What's a better way of accomplishing that?

Reply:
Yeah sorry, brain fart.